indico/web/client/js/react/components/principals/util.js

```diff
@@ -126,7 +129,7 @@ export class PrincipalType {
       [PrincipalType.multipassGroup]: Translate.string(
         'This group does not exist anymore. Please choose a different one.'
       ),
-      // event roles are hard-deleted
+      // event roles/category roles are hard-deleted
```

Comment:
"event/category roles" (or put a space around the `/` if you want to repeat the "roles") 🙈